staging/src/k8s.io/apiserver/pkg/storage/testing/utils.go

```diff
@@ -100,13 +100,13 @@ func testPropagateStore(ctx context.Context, t *testing.T, store storage.Interfa
 	return key, setOutput
 }
 
-func ExpectNoDiff(t *testing.T, msg string, expected, got interface{}) {
+func expectObjectsDeepEqual(t *testing.T, msg string, expected, actual interface{}) {
```

Comment:
I'm not entirely sure why did you rename it - can you please revert?

Reply:
yeah, naming is hard, the new name is more accurate

Reply:
okay, I will rename to `expectNoDiff` :)